src/io/flutter/logging/FlutterLogFilterPanel.form

```diff
@@ -37,6 +37,28 @@
         </constraints>
         <properties/>
       </component>
+      <component id="4ad5d" class="javax.swing.JComboBox" binding="logLevelComboBox">
+        <constraints>
+          <grid row="0" column="3" row-span="1" col-span="1" vsize-policy="0" hsize-policy="6" anchor="8" fill="0" indent="0" use-parent-layout="false">
+            <minimum-size width="100" height="-1"/>
+            <preferred-size width="100" height="-1"/>
+            <maximum-size width="100" height="-1"/>
+          </grid>
+        </constraints>
+        <properties>
+          <model>
```

Comment:
I added some model here for display purpose while design on swing form.
The real model for combo box is: https://github.com/flutter/flutter-intellij/pull/2380/files#diff-cc5515b1514ea5d6adfd3685f8d96050R38